LocalPreferences.toml

```diff
@@ -1,2 +1,3 @@
 [CUDA_Runtime_jll]
-version = "local"
+__clear__ = ["version"]
```

Comment:
Why the clear?

Reply:
I guess this is auto generated?

TBH, I'm not sure how to set the `version` here.